docs/mint.json

```diff
@@ -729,9 +729,9 @@
           "group": "Version control",
           "icon": "code-branch",
           "pages": [
-            "production/data-sync/git-sync",
+            "production/data-sync/git-integration",
             "production/data-sync/github",
-            "production/data-sync/git-integration"
+            "production/data-sync/git-sync"
```

Comment:
i purposely had them in the original order because ideally users shouldn't use the git integration feature

Reply:
if it's too confusing to have the docs two different places, can we at least prioritize the git sync docs over the git integration ones?

Reply:
No I think that's fine. We can prioritize sync

Reply:
can we have the configure git settings section live in the git sync docs? i would prefer not to link from the git sync docs to the git integration

Reply:
So you want me to merge git-integration and git-sync?

Reply:
no, in this PR you removed a lot of the stuff in the git-sync doc and replaced it with a redirect to the git-integration doc. i'm suggesting to not redirect people to the git-integration docs if possible.

Reply:
Trying not to duplicate content, since it creates multiple points for potential error.

The more I look at it, the more the structure of these docs doesn't really make sense. I'll revise